static/css/frame.scss

```diff
@@ -118,6 +118,14 @@
   padding-right: $grid-unit * 1.5;
 }
 
+.shot-fxA-signin {
```

Comment:
The uppercase 'A' here seems random, and it's not consistent with the rest of the class names.